swarm/tools/flow_studio_ui/css/flow-studio.base.css

```
@@ -1064,6 +1064,27 @@
+      top: 50%;
+      transform: translateY(-50%);
+      background: none;
+      border: none;
+      color: #9ca3af;
+      cursor: pointer;
+      padding: 0;
+      font-size: 16px;
+      line-height: 1;
+      display: none;
+      z-index: 1003;
+    }
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)